src/globalComponents/getMasterAccountId.js

```diff
@@ -1,6 +1,6 @@
 //@flow strict
 
-const MASTER_ACCOUNT_ID = '5e7ff4a9c6c3700959497009';
+const MASTER_ACCOUNT_ID = '5e93b1d0d1125f22ecd469b7';
```

Comment:
can't you actually grab it from the `.env`? The master account ID is in that file. If you can't then that's fine. I'm just uncomfortable for having the master ObjectId hardcoded here.

Reply:
I'm pretty sure we can't/wouldn't want to. If we include the env file in what we send to the frontend, then we are sending them all of our passwords/keys in plaintext too, which is bad